rancher-2.10.yaml

```diff
@@ -0,0 +1,289 @@
+      - rancher-helm3-charts
+      - rancher-kontainer-driver-metadata-${{vars.major-minor-version}}
+      - rancher-machine
+      - rancher-partner-charts
+      - rancher-rke2-charts
+      - rancher-system-charts-${{vars.major-minor-version}}
+      - rancher-telemetry
+      - rancher-ui-${{vars.major-minor-version}}
+      - runc
+      - sed
+      - shadow
+      - sysstat
+      - tini
+      - umount
+      - unzip
+      - util-linux-dev
+      - util-linux-misc
+      - vim
+      - xz
+  checks:
+    disabled:
+      - opt # docker-machine-driver-harvester should not be a compat package
+
+environment:
+  contents:
+    packages:
+      - btrfs-progs-dev
+      - cdrkit
+      - curl
+      - gpgme-dev
+
+vars:
+  py-version: 3.13
+
+var-transforms:
+  - from: ${{package.version}}
+    match: ^(\d+\.\d+)\.\d+$
+    replace: "$1"
+    to: major-minor-version
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/rancher/rancher
+      tag: v${{package.version}}
+      expected-commit: 3fdfa8c24361429803baa3fc4be739cfd4e0263a
+
+  - name: Create directories
+    runs: |
+      mkdir -p ${{targets.contextdir}}/usr/bin
+      mkdir -p ${{targets.contextdir}}/var/lib/rancher/k3s/agent/images
+      mkdir -p ${{targets.contextdir}}/var/lib/kubelet
+      mkdir -p ${{targets.contextdir}}/var/lib/cni
+      mkdir -p ${{targets.contextdir}}/var/log
+      mkdir -p ${{targets.contextdir}}/var/lib/rancher-data/driver-metadata
+      mkdir -p ${{targets.contextdir}}/var/log/auditlog
+      mkdir -p ${{targets.contextdir}}/usr/share/rancher/ui
+      mkdir -p ${{targets.contextdir}}/etc/rancher/ssl
+      mkdir -p ${{targets.contextdir}}/app/rancher
+      mkdir -p ${{targets.contextdir}}/var/lib/ca-certificates
+      mkdir -p ${{targets.contextdir}}/var/lib/rancher-data/driver-metadata
+      mkdir -p ${{targets.contextdir}}/var/lib/cattle
+      mkdir -p ${{targets.contextdir}}/opt/jail
+      mkdir -p /var/tmp
+
+  - uses: go/bump
+    with:
+      deps: |-
+        k8s.io/kubernetes@v1.31.6
+        github.com/go-jose/go-jose/v3@v3.0.4
+        golang.org/x/crypto@v0.35.0
+        golang.org/x/oauth2@v0.27.0
+        golang.org/x/net@v0.36.0
+        github.com/golang-jwt/jwt/v4@v4.5.2
+        github.com/golang-jwt/jwt/v5@v5.2.2
+
+  - uses: go/build
+    with:
+      packages: .
+      output: rancher
+      ldflags: |
+        -X main.VERSION=${{package.version}}
+        -X github.com/rancher/rancher/pkg/version.Version=${{package.version}}
+        -X github.com/rancher/rancher/pkg/version.GitCommit=$(git rev-parse --short HEAD)
+        -X github.com/rancher/rancher/pkg/settings.InjectDefaults="{\"rke-version\":\"$(grep -m1 'github.com/rancher/rke' go.mod | awk '{print $2}')\"}"
+      tags: k8s
+
+  - working-directory: hack/airgap
+    uses: go/build
+    with:
+      packages: .
+      output: export-images
+      tags: k3s_export
+
+  - name: Download pre-built rancher-tiller
```

Comment:
How will updates be handled here? I'm assuming the tiller package sources aren't available? A comment explaining this would be helpful